packages/neutron-sdk/src/bindings/msg.rs

```diff
@@ -304,6 +352,20 @@ pub enum AdminProposal {
     SoftwareUpgradeProposal(SoftwareUpgradeProposal),
     /// **CancelSoftwareUpgradeProposal** is a cancel software upgrade proposal field.
     CancelSoftwareUpgradeProposal(CancelSoftwareUpgradeProposal),
+    ///
+    UpgradeProposal(UpgradeProposal),
+    ///
+    ClientUpdateProposal(ClientUpdateProposal),
+    ///
+    PinCodesProposal(PinCodesProposal),
+    ///
+    UnpinCodesProposal(UnpinCodesProposal),
+    ///
+    SudoContractProposal(SudoContractProposal),
+    ///
+    UpdateAdminProposal(UpdateAdminProposal),
+    ///
+    ClearAdminProposal(ClearAdminProposal),
```

Comment:
To be fair I would delete these "documentation" comments anyway, they don't seem to hold any value except for stating that "x is an x".